test/test.sh

```diff
@@ -75,8 +75,8 @@ stopit test.pid
 stopit pgdata/postmaster.pid
 
 mkdir -p $LOGDIR
-rm -f $BOUNCER_LOG $PG_LOG
-rm -rf $PGDATA
+echo > $BOUNCER_LOG
```

Comment:
explain please

Reply:
My mistake including this in the commit. I will fix this.